src/web/app/pages-instructor/instructor-course-student-details-page/instructor-course-student-details-page.component.html

```diff
@@ -1,3 +1,128 @@
-<p>
-  instructor-course-student-details-page works!
-</p>
+<ng-template #notSpecified><i class="text-muted">Not Specified</i></ng-template>
+<div *ngIf="student">
+  <h1>{{student.name}}</h1>
+  <div class="row" *ngIf="studentProfile">
+    <div class="col-12">
+      <div class="row">
+        <div class="col-md-2 col-3">
+          <img src="{{getPictureUrl(studentProfile.pictureKey)}}" class="profile-pic float-right">
+        </div>
+        <div class="col-md-10 col-sm-9 col-8">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th colspan="2"><h4><span class="font-weight-bold">Profile Details </span>(submitted by student)</h4></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td class="font-weight-bold">Short Name (Gender)</td>
+                <td><span *ngIf="studentProfile.shortName; else notSpecified">{{studentProfile.shortName}}</span>
+                  (<i *ngIf="studentProfile.gender != 'other'; else notSpecified">{{studentProfile.gender}}</i>)
+                </td>
+              </tr>
+              <tr>
+                <td class="font-weight-bold">Personal Email</td>
+                <td><span *ngIf="studentProfile.email; else notSpecified">{{studentProfile.email}}</span>
+              </tr>
+              <tr>
+                <td class="font-weight-bold">Institution</td>
+                <td><span *ngIf="studentProfile.institute; else notSpecified">{{studentProfile.institute}}</span>
+              </tr>
+              <tr>
+                <td class="font-weight-bold">Nationality</td>
+                <td><span *ngIf="studentProfile.nationality; else notSpecified">{{studentProfile.nationality}}</span>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="card border-primary">
+    <div class="card-body fill-plain">
+      <div class="form form-horizontal">
+        <div class="form-group">
+          <div class="col-sm-12">
+            <h4 class="font-weight-bold">Enrollment Details</h4>
+          </div>
+        </div>
+        <div class="form-group student-info-row row">
+          <div class="col-4 text-right">
+            <label>Course</label>
+          </div>
+          <div class="col-8">
+            <a routerLink="/web/instructor/courses/details?courseid={{student.course}}">
```

Comment:
Adding key-value query parameters to `routerLink` is not done this way: refer [here](https://angular.io/api/router/RouterLink).